vortex-array/src/compute/filter.rs

```diff
@@ -202,6 +204,10 @@ impl FilterMask {
         self.array.len() - self.true_count
     }
 
+    pub fn true_range(&self) -> (usize, usize) {
```

Comment:
we got rid of true_range in #1468, prob want to remove here as well

Reply:
Yup, I’m working on fixing the last issue today and have it rebased locally